src/main/kotlin/no/nav/familie/klage/brev/BrevInnhold.kt

```diff
@@ -170,7 +170,7 @@ object BrevInnhold {
         Stønadstype.OVERGANGSSTØNAD,
         Stønadstype.BARNETILSYN,
         Stønadstype.SKOLEPENGER,
-        -> "klage.nav.no/familie/enslig-mor-eller-far"
+        -> "nav.no/klage#overgangsstonad-til-enslig-mor-eller-far"
```

Comment:
Men de klager vel ikke alltid på overgangsstønad?

Reply:
Barnetilsyn bør lenke til; https://www.nav.no/klage#stonad-til-barnetilsyn-for-enslig-mor-eller-far
Skolepenger bør lenke til: https://www.nav.no/klage#stonad-til-skolepenger-for-enslig-mor-eller-far

Reply:
https://www.nav.no/klage#barnetrygd
osv